.github/workflows/publish-to-npm.yaml

```diff
@@ -0,0 +1,33 @@
+name: Build and publish new version of @intersect.mbo/pdf-ui package
+run-name: Publish by @${{ github.actor }}
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: pdf-ui
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
```

Comment:
@nebojsact Please set the needed node version

Reply:
Node version is ok, we just need NPM_TOKEN :)